src/pybind/mgr/dashboard/module.py

```diff
@@ -174,6 +148,32 @@ def _configure(self):
         }
 
         if ssl:
+            # SSL initialization
```

Comment:
Awesome! Could you please then also rename the class `SSLCherryPyConfig` to `CherryPyConfig`, as its no longer purely for SSL initialization. ? Do you think it would make sense to separate this into a new method `_configure_ssl`?

Reply:
I agree, this would be appropriate. @b-ranto - can you please include that modification before we merge this?